PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -283,9 +283,15 @@ def miniAOD_customizeCommon(process):
     )
     task.add(process.patJetPuppiCharge)
 
+    ## Using Puppi candidates as input for b tagging: just in Phase 2
+    _pfCandidates = 'particleFlow'
+    from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+    if process.isUsingModifier( phase2_common ):
```

Comment:
There must be a better way to do this... @arizzi @gpetruc is there a best practice for Eras in miniAOD customization functions?